src/api/exportData/exportDataClient.js

```
@@ -0,0 +1,53 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2021 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+import JsFileDownloader from "js-file-downloader";
+import * as Sentry from "@sentry/react";
+
+export default async function exportDataClient(
```

given its brittleness I kind of wish this function were ... not public, or named differently, or that it would actually throw an error or something for the browsers it is known not to support instead of just failing silently if it were actually to be used incorrectly? i am not totally sure though which, if any, of those options is best here

---

I updated this to `exportZipDataOnMobileDevices` and added an error if it's not used for a mobile user - let me know what you think and if that helps.